explainer.md

```diff
@@ -62,55 +65,55 @@ The purchase flow itself uses the [Payment Request API](https://w3c.github.io/pa
 
 ```js
 new PaymentRequest(
-  [{supportedMethods: "https://example.com/billing",
+  [{supportedMethods: 'https://example.com/billing',
     data: {itemId: item.id}}]);
 ```
 
 Note that as part of this proposal, we are proposing to [remove the requirement](https://github.com/w3c/payment-request/issues/912) of the `total` member of the `details` dictionary, since the source of truth for the item price (that will be displayed to the user in the purchase confirmation dialog) is known by the server, based on the item ID. The exact format of the `data` member is up to the store (the spec simply says this is an `object`). Some stores may allow multiple items to be purchased at the same time, others only a single item.
 
 ### Acknowledging a purchase
 
-Some stores will require that the user acknowledge a purchase once it has succeeded. In this case, the payment response will return a `PurchaseToken`, which can be used with the `consume` and `acknowledge` methods.
+Some stores will require that the user acknowledge a purchase once it has succeeded. In this case, the payment response will return a `PurchaseToken`, which can be used with the `acknowledge` method.
 
-Items that are designed to be purchased multiple times must be consumed. An example of a consumable purchase is an in-game powerup that makes the player stronger for a short period of time. Once it is consumed, it can be purchased again.
+Items that are designed to be purchased repeatedly must be acknowledged with the `repeatable` flag. An example of a repeatable purchase is an in-game powerup that makes the player stronger for a short period of time. Once it is acknowledged, it can be purchased again.
```

Comment:
I feel like changing "multiple times" to "repeatedly" makes it less clear what this means. (It might imply a subscription?) In general, using a different phrasing in the prose text to the official name helps to clarify things instead of just being redundant.

You could say "more than once" instead of "multiple times", optionally.

Reply:
Also say "once it is acknowledged with the `repeatable` flag" just to be clear.

Reply:
Done.
Switched back to "multiple times".
Added "with the repeatable flag" and below, "with the onetime flag" in the corresponding sentence.